modules/ossm-cluster-wide-migration-methods.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+
+// * service-mesh-docs-main/migrating/cluster-wide/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-cluster-wide-migration-methods_{context}"]
+= Cluster-wide migration methods
+
+Migrating the control plane from {SMProductName} {SMv2Version} to {SMProduct} 3.0 can be performed by using one of the following methods:
```

Comment:
```suggestion
You can migrate the control plane from {SMProductName} {SMv2Version} to {SMProduct} 3.0 by using one of the following methods:
```
- To better engage the reader, IBM Style advises writing in second person: https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#second-person

Reply:
done